application/client/src/app/service/session/session.ts

```diff
@@ -68,6 +68,17 @@ export class Session extends Base {
                 },
             },
         });
+        this._toolbar.add({
+            name: 'Chart',
+            active: false,
+            closable: false,
+            content: {
+                factory: components.get('app-views-chart'),
+                inputs: {
+                    session: this,
+                },
+            },
```

Comment:
`uuid` field is missed
you can add uuid into `application/client/src/app/service/session/session.ts`:
```
const TOOLBAR_TAB_SEARCH = unique();
const TOOLBAR_TAB_PRESET = unique();
const TOOLBAR_TAB_CHART = unique();

```